python/fbprophet/forecaster.py

```diff
@@ -1474,3 +1505,57 @@ def copy(self, cutoff=None):
             DeprecationWarning,
         )
         return prophet_copy(m=self, cutoff=cutoff)
+
+    def make_holidays(self, year_list, country_list):
```

Comment:
This doesn't actually use `self`, so let's go ahead and make it not a method of Prophet (I've been trying to reduce the number of methods on Prophet since it is quite long). It could either go in `hdays` and be imported from there, or we could make a new `utils.py` file that it could go in and be imported from.